src/test/java/org/folio/circulation/support/http/client/CqlQueryTests.java

```diff
@@ -106,6 +107,14 @@ public void canApplyGreaterThenOperator() {
     assertThat(query.value().asText(), is(format("lastTime>\"%s\"", dateTime)));
   }
 
+  @Test
+  public void canApplyNotEqualOperator() {
+    DateTime dateTime = now(UTC);
+    Result<CqlQuery> query = notEqual("lastTime", dateTime);
+
+    assertThat(query.value().asText(), is(format("lastTime<>\"%s\"", dateTime)));
```

Comment:
Why are we only testing it with DateTime value? In the code I believe we were passing "[]" to notEqual().